### PR TITLE
add spinner when open positions details cell

### PR DESCRIPTION
**Pull request by guillaumedebavelaere**
This was required for the fidelity demo and was on a separate branch.
capture d ecran 2017-08-25 a 16 49 18

### CODE REVIEW

File: TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsTableViewManager.swift [hunk 1]
```diff
@@ -70,15 +70,23 @@ class TradeItPortfolioAccountDetailsTableViewManager: NSObject, UITableViewDeleg
         } else if self.selectedPositionIndex != -1 {
             let prevPath = IndexPath(row: self.selectedPositionIndex, section: SECTIONS.positions.rawValue)
             self.selectedPositionIndex = indexPath.row
-            self.positions?[self.selectedPositionIndex].refreshQuote(onFinished: {
-                self.reloadTableViewAtIndexPath([prevPath, indexPath])
-            })
+            if let cell = self.table?.cellForRow(at: indexPath) as? TradeItPortfolioEquityPositionsTableViewCell {
```
Comment (jsom) [inline TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsTableViewManager.swift]: Better to use `guard` here

**Comment by jsom** [comment]
@guillaumedebavelaere I rebased this. Was there a design for this? Having it behind the content seems a little weird.

**Comment by guillaumedebavelaere** [comment]
@jsom no design was done for this

**Comment by jsom** [comment]
OK - let's merge this for now but can you put in a story in for Ameer to create a design for it so we can make a second pass then pass it to Betsy to get prioritised. Thanks!

**Pull request merged**